docker/relayerConfigTemplate.json

```diff
@@ -5,7 +5,7 @@
     "source-subnets": [
         {
             "subnet-id": "11111111111111111111111111111111LpoYY",
-            "chain-id": "<C_CHAIN_CHAIN_ID>",
+            "blockchain-id": "<C_CHAIN_CHAIN_ID>",
```

Comment:
TODO: Update the AWM Relayer version in versions.sh once https://github.com/ava-labs/awm-relayer/pull/115 is merged and a new release created.

Reply:
Nit, should we renaming the template variables `<BLOCKCHAIN_ID>` rather than `<CHAIN_ID>`?